src/Export/Scripts/mods.lua

```diff
@@ -174,5 +174,9 @@ writeMods("../Data/ModNecropolis.lua", function(mod)
 	return mod.Domain == 1 and mod.Id:match("^NecropolisCrafting")
 end)
 
+writeMods("../Data/BeastCraft.lua", function(mod)
+    return (mod.Id:match("Aspect")  and mod.GenerationType == 2)  -- Aspect Crafts
```

Comment:
Hi, I'm the officially unofficial imaginary mascot & chief pedant of PoB, and this line should be TAB indented. 😁